cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -1744,11 +1775,28 @@
                 300000L,
                 ConfigDef.Importance.LOW,
                 ANOMALY_DETECTION_INTERVAL_MS_DOC)
+        .define(GOAL_VIOLATION_DETECTION_INTERVAL_MS_CONFIG,
+                ConfigDef.Type.LONG,
+                ConfigDef.Importance.LOW,
+                GOAL_VIOLATION_DETECTION_INTERVAL_MS_DOC)
+        .define(METRIC_ANOMALY_DETECTION_INTERVAL_MS_CONFIG,
+                ConfigDef.Type.LONG,
+                ConfigDef.Importance.LOW,
+                METRIC_ANOMALY_DETECTION_INTERVAL_MS_DOC)
+        .define(DISK_FAILURE_DETECTION_INTERVAL_MS_CONFIG,
+                ConfigDef.Type.LONG,
+                ConfigDef.Importance.LOW,
+                DISK_FAILURE_DETECTION_INTERVAL_MS_DOC)
         .define(ANOMALY_DETECTION_ALLOW_CAPACITY_ESTIMATION_CONFIG,
                 ConfigDef.Type.BOOLEAN,
                 DEFAULT_ANOMALY_DETECTION_ALLOW_CAPACITY_ESTIMATION_CONFIG,
                 ConfigDef.Importance.LOW,
                 ANOMALY_DETECTION_ALLOW_CAPACITY_ESTIMATION_DOC)
+        .define(BROKER_FAILURE_DETECTION_BACKOFF_MS_CONFIG,
+                ConfigDef.Type.LONG,
+                300000L,
```

Comment:
Nit: Can we move the hardcoded value to a static variable?

Reply:
There are a lot of hard-coded default values in config definition.
 I think we'd better change all of them in a separate patch.